packages/web-components/fast-foundation/src/badge/README.md

```diff
@@ -0,0 +1,13 @@
+# Usage
+```ts
+import { customElement } from "@microsoft/fast-element";
+import { Badge, BadgeTemplate as template } from "@microsoft/fast-foundation";
+import { BadgeStyles as styles } from "./badge.styles";
+
+@customElement({
+    name: "fast-badge",
+    template,
+    styles,
+})
+export class FASTBadge extends Badge {}
```

Comment:
just a note that we probably need to expand on badge with the inclusion of css variables (not necessarily in this PR but without that the usage is incomplete).